cellbase-lib/src/main/java/org/opencb/cellbase/lib/variant/annotation/ConsequenceTypeCalculator.java

```diff
@@ -406,6 +404,12 @@ protected boolean setInsertionAlleleAminoacidChange(String referenceCodon, char[
         return firstCodon;
     }
 
-
+    // determine the consequence type source (the geneset) based on the gene ID
+    protected String getSource(String geneId) {
+        if (!geneId.startsWith("ENS")) {
+            return "refseq";
```

Comment:
Shouldn't we use a centralised enum for the source? This is going to be very stable (actually unlikely to ever change), and would ensure consistency in different parts of the code (WS, Managers, Adaptors, ...)